### DISCUSSION TITLE
User Management changes

### DISCUSSION THREAD

**Discussion by mrantivirus** [gis-ops/docker-valhalla#64] [Q&A] [answered]
Hi. I was using version 3.1.4 of the Docker image. I tried running with Australia, Asia, North America and Europe PBF files, but for some reason I kept getting a core dump when it starts processing "layer 7" (sorry, I don't have the logs anymore). So I tried updating to `gisops/valhalla:latest` assuming maybe some upstream fixes would help. However now, I can't seem to get past the following error.
```
INFO: Running container with user valhalla UID 59999 and GID 59999.

ERROR: If you run with custom UID or GID you have to create the mapped directory to the container's /custom_files manually before starting the image
```

I followed the guide and run this on my local machine and on a hosted server.
```
docker pull ghcr.io/gis-ops/docker-valhalla/valhalla:latest
// Tried docker pull gisops/valhalla:latest as well
docker run -dt --name valhalla_gis-ops -p 8002:8002 -v $PWD/custom_files:/custom_files gisops/valhalla:latest
```

Am I missing a step or do I need to manually build the image myself? Thanks in advance.

**Comment by nilsnolde** [comment, marked as answer]
argh, yeah, seems there was a very awkward bug.. so this would've deserved to be an issue instead:)

I just posted a fix, both dockerhub and github packages should be updated with a new `latest` image, so it shouldn't matter which one you use. (they should always get updated at the same time)

**Reply by mrantivirus** [reply]
Thanks for the quick fix! Yeah, I guess I should have written it in `issues`. I wasn't sure if I was missing something or if it was a bug on your end. Either way, the container is running and building tiles now.
